cloud/google/cmd/gce-machine-controller/Makefile

```diff
@@ -18,7 +18,7 @@ GCR_BUCKET = k8s-cluster-api
 PREFIX = gcr.io/$(GCR_BUCKET)
 DEV_PREFIX ?= gcr.io/$(shell gcloud config get-value project)
 NAME = gce-machine-controller
-TAG = 0.0.9
+TAG = 0.0.10
```

Comment:
Can you update this to 0.0.11? I just built 0.0.10 for #183, which I'm hoping to merge this morning.

Reply:
Yes, will do.